tests/test_job.py

```diff
@@ -0,0 +1,111 @@
+from typing import Any, Tuple
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+from pytest_mock import MockerFixture
+from qiskit.circuit import QuantumCircuit
+
+from qiskit_quantuminspire.qi_jobs import QIJob
+
+
+@pytest.fixture()
+def api_mock(mocker: MockerFixture) -> Tuple[MagicMock, MagicMock]:
```

Comment:
This mock has some overlap with the one in test_qi_provider.

Reply:
moved to a common fixture in conftest.py

Reply:
This can then be removed right?

Reply:
Missing push. This file will actually dissappear and move everything to test_jobs.py.